typeDefs.js

```diff
@@ -334,6 +334,7 @@ const typeDefs = gql`
     updateUserByToken(user: UserInput!): User
     confirmUser(token: String!): Boolean
     recoverUser(email: String!, language: Int!): Boolean
+    switchUserStatus(id: ID!): Boolean
```

Comment:
Also if you perform check in the service and can throw an error you need to use union type `Boolean | Error`. Look at `newsResolver` as an example of error handling and change resolver and schema

Reply:
did it